imi/src/main/java/org/motechproject/nms/imi/domain/FileProcessedStatus.java

```diff
@@ -4,23 +4,29 @@
  * File processing status, provided by IVR
  */
 public enum FileProcessedStatus {
-    FILE_PROCESSED_SUCCESSFULLY(8000),
-    FILE_NOT_ACCESSIBLE(8001),
-    FILE_CHECKSUM_ERROR(8002),
-    FILE_RECORDSCOUNT_ERROR(8003),
-    FILE_OUTSIDE_SOCIAL_HOURS(8004),
-    FILE_ERROR_IN_FILE_FORMAT(8005);
+    FILE_PROCESSED_SUCCESSFULLY(8000, "FILE_PROCESSED_SUCCESSFULLY"),
+    FILE_NOT_ACCESSIBLE(8001, "FILE_NOT_ACCESSIBLE"),
+    FILE_CHECKSUM_ERROR(8002, "FILE_CHECKSUM_ERROR"),
+    FILE_RECORDSCOUNT_ERROR(8003, "FILE_RECORDSCOUNT_ERROR"),
+    FILE_OUTSIDE_SOCIAL_HOURS(8004, "FILE_OUTSIDE_SOCIAL_HOURS"),
+    FILE_ERROR_IN_FILE_FORMAT(8005, "FILE_ERROR_IN_FILE_FORMAT");
 
     private final int value;
+    private final String name;
 
-    FileProcessedStatus(int value) {
+    FileProcessedStatus(int value, String name) {
         this.value = value;
+        this.name = name;
     }
 
     public int getValue() {
         return value;
     }
 
+    public String getName() {
+        return name;
+    }
+
     public static boolean isValid(int i) {
```

Comment:
When I read the method isValid I first thought it was returning if the File was valid based off of the status code.  Not a big deal, but I wonder if anyone else will confuse the enum itself being valid with the enum signifying the file is valid.